package tryCatch;

public class ThrowsException {

	public static void main(String[] args) {
		throw new RuntimeException(); // 강제로 예외 발생시키기 - 테스트 할 때 사용할 수 있음
	}

}
